core/field_textinput.js

```diff
@@ -275,7 +275,7 @@ Blockly.FieldTextInput.prototype.showInlineEditor_ = function(quietInput) {
  *     which event handlers will be bound.
  * @private
  */
-Blockly.FieldTextInput.prototype.bindEvents_ = function(htmlInput) {
+Blockly.FieldTextInput.prototype.bindInputEvents = function(htmlInput) {
```

Comment:
If this is private, add an undescore.

Reply:
Derp, Fixed

Reply:
Still need to remove the @private tag :)

Reply:
bindInputEvents_ is different than bindEvents_. bindInputEvents_ is just used by the textInput for its htmlInput, while bindEvents_ is for binding to the field proper. Maybe they need better names?

Reply:
It's fine--I was looking at an outdated copy of the code there.